chain/chunks/src/lib.rs

```diff
@@ -1556,7 +1556,7 @@ impl ShardsManager {
 
         // 2. check protocol version
         let protocol_version = self.runtime_adapter.get_epoch_protocol_version(&epoch_id)?;
-        if !header.version_range().contains(protocol_version) {
```

Comment:
Looking at the call-site, perhaps we don't need a range here at all and just `header.valid_for(protocol_version)` ?